test/model/store_test.dart

```diff
@@ -1071,7 +1071,7 @@ void main() {
       if (defaultTargetPlatform == TargetPlatform.android) {
         checkLastRequestFcm(token: '012abc');
       } else {
-        checkLastRequestApns(token: '012abc', appid: 'com.zulip.flutter');
+        checkLastRequestApns(token: '012abc', appid:'com.zulip.flutter.test');
```

Comment:
nit: space before the string
```suggestion
        checkLastRequestApns(token: '012abc', appid: 'com.zulip.flutter.test');
```